### ISSUE TITLE
Add link to your public Galaxy History

### ISSUE THREAD

**Issue by annefou** [DataRockMyWorld/DataRockMyWorld.github.io#8]
One last thing: could you add the link to your Galaxy history on your website?

**Comment by annefou** [comment]
And add your Galaxy history as a contribution to outreachy. I don't think I have seen the history with ERA5 data analysis.

**Comment by annefou** [comment]
@DataRockMyWorld and once added you can submit your contribution to outreachy. Well done! 🎉

**Comment by DataRockMyWorld** [comment]
Link updated on website. 
Thank you.
I can still work on the website after submitting it right? I want to improve the aesthetics in my spare time.
And I think galaxy should get an icon added to font awesome icons for easy reference on websites 😊

**Comment by annefou** [comment]
> Link updated on website. Thank you. I can still work on the website after submitting it right? I want to improve the aesthetics in my spare time. And I think galaxy should get an icon added to font awesome icons for easy reference on websites 😊

Yes you can still continue working on your website!  I see how much progress you have done. Great job!

**Comment by DataRockMyWorld** [comment]
Hi Anne,
Find attached please.
https://datarockmyworld.github.io/Ghana-Climate-Change/
Regards,
Jewel Bansah 

    On Wednesday, April 6, 2022, 09:39:50 PM GMT, Anne Fouilloux ***@***.***> wrote:  
 
 



Link updated on website. Thank you. I can still work on the website after submitting it right? I want to improve the aesthetics in my spare time. And I think galaxy should get an icon added to font awesome icons for easy reference on websites 😊
 

Yes you can still continue working on your website! I see how much progress you have done. Great job!

**Comment by annefou** [comment]
This is very interesting! 
In the last section, could you think about a concrete example on how Galaxy Climate could support local initiatives (from those mentioned in the paragraph "Climate change initiatives in Ghana"). For instance, when trees are planted, is there any monitoring in place to assess the effectiveness of this action? Do they collect data so that in 10-20 years, we would see an improvement?

In short, what would you suggest to do as part of the Outreachy internship? This is also to help you to define a timeline (that will be asked when you submit your final application).

**Comment by DataRockMyWorld** [comment]
Thank you for the feedback.I will zero down on specifics.I have an interview scheduled with an officer of the climate change section of Ghana Meteorological Agency tomorrow.I hope to have an idea of the tools they use at the moment and how best to employ Galaxy in our current processes.
Regards,
Jewel Bansah 

    On Tuesday, April 12, 2022, 11:31:52 AM GMT, Anne Fouilloux ***@***.***> wrote:  
 
 


This is very interesting!
In the last section, could you think about a concrete example on how Galaxy Climate could support local initiatives (from those mentioned in the paragraph "Climate change initiatives in Ghana"). For instance, when trees are planted, is there any monitoring in place to assess the effectiveness of this action? Do they collect data so that in 10-20 years, we would see an improvement?

In short, what would you suggest to do as part of the Outreachy internship? This is also to help you to define a timeline (that will be asked when you submit your final application).
